internal/pkg/template/repository/manager.go

```diff
@@ -94,7 +94,7 @@ func pullRepo(ctx context.Context, logger log.Logger, repo *git.Repository) erro
 
 	newHash, err := repo.CommitHash(ctx)
 	if err != nil {
-		return err
+		logger.Errorf("cannot get new commit hash", err)
```

Comment:
co v takom pripade obsahuje `newHash` premenna ak `err != nil` ? Nerozbije sa to dalej v tom kode kedze to nerobi return?

Reply:
alebo tym ze je nizsie podmienka `oldHash == newHash` sa len urobi `return nil`?

Reply:
V `newHash` bude prázdný string. No jako říkal jsem si že to ničemu tak nevadí, ale máš pravdu, přidal jsem tam return. 🙂 

Ale ten první `oldHash, err := repo.CommitHash(ctx)` bych neukončoval. Pokud je chyba už v něm, tak bych chtěl vidět jak dopadne ten `repo.Pull()` potom. (To že vyfailuje nijak funkci nevadí, získává se jen kvůli tomu aby se vypsalo jestli došlo ke změně nebo ne.) Divný je, že ta chyba vypisuje do PT jen `"cannot get repository hash"` a nic jinýho. Podle https://github.com/keboola/keboola-as-code/blob/aa144c97e78fc40646e1c25ef62c4539c13669c7/internal/pkg/git/git.go#L133-L133 by se tam měl vypsat i obsah `stderr`, ale v něm teda asi nic není, tak se nemám moc čeho chytit.